metadata_fetcher/fetch_registry_collections.py

```diff
@@ -27,7 +27,16 @@ def fetch_endpoint(url, limit=None):
         for collection in collections:
             if limit and len(results) >= limit:
                 break
+
             log_msg = f"[{collection['collection_id']}]: " + "{}"
+
+            if not collection["ready_for_publication"]:
+                print(log_msg.format(
+                    f"Skipping collection {collection['collection_id']} "
+                    "(not ready for publication)"
+                ))
+                continue
```

Comment:
I really think this should actually just get added to the registry url that you provide as input to this script. For example: https://registry.cdlib.org/api/v1/rikoltifetcher/?format=json&mapper_type=up_oai_dc&ready_for_publication=true

The whole point of the harvester is to fetch and map collections not ready for publication. Then, when they have been QAed and we are certain the mappings are correct and the contributing institution is happy with how their records look, we mark them as ready for publication and push the records into our published index. 

In your case 'ready for publication = False' is doubling as a 'not yet ready for harvesting' flag. Collections can wind up in this state for all kinds of reasons, even while the Collection record still persists in the Collection Registry. 

I think it makes more sense to add `ready_for_publication=true` to the API URL passed to this script in those cases where you want to skip testing certain collections.